schema.go

```diff
@@ -340,16 +341,16 @@ func (indexField *IndexField) DecodeMsgpack(d *msgpack.Decoder) error {
 	return errors.New("unexpected schema format (index fields)")
 }
 
-func (conn *Connection) loadSchema() (err error) {
+func GetSchema(conn *Connection) (*Schema, error) {
```

Comment:
We could use a common interface here.
```suggestion
func GetSchema(conn Connector) (*Schema, error) {
```

Reply:
Updated.